### PR TITLE
Updating Board Minutes (March)

### PR DESCRIPTION
**Pull request by martha-thomae**
Add 2020-03-25 board minutes.

### CODE REVIEW

**Comment by martha-thomae** [comment]
@bwbohl would you mind just taking a quick look before I merge?

File: community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	- The idea is to have the papers ready in preparation for the conference (this can be indicated in the email). Being able to read the papers beforehand will help presenters to keep their presentations short—now that they have just 3 minutes to present—and go directly to the discussion.
+- A few suggestions about the proceedings:
+	- Word template for ISMIR (is ready)
+	- People to send the PDF not the Word document
+	- Just a set of guidelines in the email asking for the conference papers (font, size, spacing, citation guidelines)
+	- We have only 8 weeks, so we shouldn’t design a template now. Again, we can use a pre-existing Word template (e.g., ISMIR) and send PDF (no editing to be expected).
+
+## MEC 2021
+
+- Stefan Münnich will be the chair for next year
```
Comment (bwbohl) [inline community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md:43]: program chair?

File: community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- This section can be added by the “chapter editor” by quickly listing the contributors according to their contributions (see commit history)
+
+**The last two topics can be discussed in further detail in the ODD Friday**
+
+### Copyright
+
+This is for the images used in the guidelines (when they come from manuscripts):
+
+- At the beginning of the guidelines, a statement: “We consider the use of images as fair use.”
+- MEI falls into German law, 15% of copyright work that can be used. So, very small snippets are all right.
```
Comment (bwbohl) [inline community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md:84]: I would not be too sure about the applicable law

Comment (martha-thomae) [reply community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md:84]: This is what Johannes said. I can just "cite" it.

File: community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md (hunk 1 above)
Comment (martha-thomae) [reply community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md:43]: Will change it now.

**Comment by martha-thomae** [comment]
How about now, @bwbohl?

File: community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+- This section can be added by the “chapter editor” by quickly listing the contributors according to their contributions (see commit history)
+
+**The last two topics can be discussed in further detail in the ODD Friday**
+
+### Copyright
+
+This is for the images used in the guidelines (when they come from manuscripts):
+
+- At the beginning of the guidelines, a statement: “We consider the use of images as fair use.”
+- According to Johannes: MEI falls into German law, 15% of copyright work that can be used. So, very small snippets are all right.
```
Comment (bwbohl) [inline community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md:84]: > According to Johannes: MEI is most likely to fall into German jurisdiction,…

what do you think?

Comment (martha-thomae) [reply community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md:84]: Good!

**Comment by martha-thomae** [comment]
Hi @bwbohl! So, I see "changes approved" (I don't know how long this has been here). Anyway, so I guess I can merge this already? I just wanted to double-check.

**Pull request merged**